doc/radosgw/lua-scripting.rst

```diff
@@ -0,0 +1,338 @@
+=============
+Lua Scripting
+=============
+
+.. versionadded:: Pacific
+
+.. contents::
+
+This feature allows users to provide a Lua script that will be executed after a request has been processed.
```

Comment:
Suggestion: 

This feature allows users to upload a Lua script that runs after the user's request has been processed. The script can access fields in the request and to modify some fields. All Lua language features can be used in the script.